[DOMException-constructor.html]
  type: testharness
  [new DOMException()]
    expected: FAIL

  [new DOMException(): own-ness]
    expected: FAIL

  [new DOMException(null)]
    expected: FAIL

  [new DOMException(undefined)]
    expected: FAIL

  [new DOMException(undefined): own-ness]
    expected: FAIL

  [new DOMException("foo")]
    expected: FAIL

  [new DOMException("foo"): own-ness]
    expected: FAIL

  [new DOMException("bar", undefined)]
    expected: FAIL

  [new DOMException("bar", "NotSupportedError")]
    expected: FAIL

  [new DOMException("bar", "NotSupportedError"): own-ness]
    expected: FAIL

  [new DOMException("bar", "foo")]
    expected: FAIL

  [new DOMexception("msg", "IndexSizeError")]
    expected: FAIL

  [new DOMexception("msg", "HierarchyRequestError")]
    expected: FAIL

  [new DOMexception("msg", "WrongDocumentError")]
    expected: FAIL

  [new DOMexception("msg", "InvalidCharacterError")]
    expected: FAIL

  [new DOMexception("msg", "NoModificationAllowedError")]
    expected: FAIL

  [new DOMexception("msg", "NotFoundError")]
    expected: FAIL

  [new DOMexception("msg", "NotSupportedError")]
    expected: FAIL

  [new DOMexception("msg", "InUseAttributeError")]
    expected: FAIL

  [new DOMexception("msg", "InvalidStateError")]
    expected: FAIL

  [new DOMexception("msg", "SyntaxError")]
    expected: FAIL

  [new DOMexception("msg", "InvalidModificationError")]
    expected: FAIL

  [new DOMexception("msg", "NamespaceError")]
    expected: FAIL

  [new DOMexception("msg", "InvalidAccessError")]
    expected: FAIL

  [new DOMexception("msg", "SecurityError")]
    expected: FAIL

  [new DOMexception("msg", "NetworkError")]
    expected: FAIL

  [new DOMexception("msg", "AbortError")]
    expected: FAIL

  [new DOMexception("msg", "URLMismatchError")]
    expected: FAIL

  [new DOMexception("msg", "QuotaExceededError")]
    expected: FAIL

  [new DOMexception("msg", "TimeoutError")]
    expected: FAIL

  [new DOMexception("msg", "InvalidNodeTypeError")]
    expected: FAIL

  [new DOMexception("msg", "DataCloneError")]
    expected: FAIL

  [new DOMException("bar", "UnknownError")]
    expected: FAIL

